packages/fast-usdc/src/fast-usdc.contract.js

```diff
@@ -87,6 +83,13 @@ export const contract = async (zcf, privateArgs, zone, tools) => {
   );
 
   const creatorFacet = zone.exo('Fast USDC Creator', undefined, {
+    /**
+     * @param {string} oracleId
+     */
+    makeOracleInvitation(oracleId) {
```

Comment:
this matches the shape in https://github.com/Agoric/agoric-sdk/pull/10478/files#diff-6844e04a4f3569538518f0703c16d6a34acb5ca95fcb04d566bb67b3524da736R97-R100

Reply:
yeah; I peeked :)